src/Card/Card.tsx

```diff
@@ -1,28 +1,40 @@
 import React from 'react';
 import { Link } from 'react-router-dom'
-
 import './Card.scss'
 
 interface CardProps {
-  name: string
+  city: {
+    name: string, 
+    temp: number,
+    aqi: number,
+    pollen: number,
+    fire: string
+  }
 }
 
 const Card: React.FC<CardProps> = (props) => {
   return (
-    <Link
-      to={`/${props.name}`}
-      className="card-link"
-      style={{ textDecoration: "none" }}
-    >
-      <article className="card-container" key={props.name}>
-        <h2 className="card-header">{props.name}</h2>
-        <h3 className="card-temp">35&deg;</h3>
-        <p className="card-air">AQI</p>
-        <p className="card-pollen">🐝</p>
-        <p className="card-hazard">🔥</p>
-      </article>
-    </Link>
-  )
+   <Link
+    to={`/${props.city.name}`}
+    className="card-link-wrapper"
+    style={{ textDecoration: "none" }}
+   >
+    <article className="card-container" key={props.city.name}>
+     <h2 className="card-header">{props.city.name}</h2>
+     <div className='card-air-temp'>
+      <h3 className="card-temp">{props.city.temp}&deg;F</h3>
+      <h3 className="card-air"><b>{props.city.aqi}</b> </h3> 
```

Comment:
I am going to use the names you used as keys in the data structure I am building that way, once its incorporated it will be a simple transition from hard coded data to the dynamic data.